acm/registration/odh-core.application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: odh-core
+  namespace: rhoai-edge-acm
```

Comment:
Should there be a Namespace resource before this, to create this namespace?

Reply:
Corrected. Since it was copied/pasted directly from the ACM Console UI which creates the NS automatically